Dockerfile

```diff
@@ -1,11 +1,7 @@
 FROM golang:1.15.2
-#RUN INSTALL_PKGS="golang" && \
-#    dnf install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
-#    rpm -V $INSTALL_PKGS && \
-#    dnf clean all -y
-RUN mkdir /tmp/src
-COPY . /tmp/src
-RUN cd /tmp/src; make build
-ENTRYPOINT /tmp/src/sippy
-
+WORKDIR /go/src/sippy
+COPY . .
+RUN make build
+ENTRYPOINT ["/go/src/sippy/sippy"]
```

Comment:
fetchdata.sh and fetchdata-kube.sh will need to be updated for this.  and the deploymentconfig yamls

Reply:
updated